GameWorld/View3D/SceneNodes/Rmv2ModelNodeLoader.cs

```diff
@@ -46,6 +46,22 @@ public void CreateModelNodesFromFile(Rmv2ModelNode outputNode, RmvFile model, An
                 {
                     var geometry = MeshBuilderService.BuildMeshFromRmvModel(model.ModelList[lodIndex][modelIndex], model.Header.SkeletonName, _contextFactory.Create());
                     var rmvModel = model.ModelList[lodIndex][modelIndex];
+
```

Comment:
when looking for empty string, prefer using string.IsNullOrWhiteSpace as it covers more cases. Typically you want to cover them all when you are doing == "".